components/script/dom/blob.rs

```diff
@@ -33,17 +33,18 @@ pub struct FileBlob {
 }
 
 
-/// Blob backend implementation
+/// Differen backends of Blob
 #[must_root]
 #[derive(JSTraceable)]
 pub enum BlobImpl {
-    /// File-based blob
+    /// File-based blob, whose content lives in net process
     File(FileBlob),
-    /// Memory-based blob
+    /// Memory-based blob, whose content lives in script process
```

Comment:
No, I meant that "script process" here should be "the script process", and the same goes for "net process" above